### DISCUSSION TITLE
What is the connection between this repo and prometheus-community/helm-charts

### DISCUSSION THREAD

**Discussion by petertakacs** [prometheus-operator/kube-prometheus#1509] [Q&A] [answered]
This repo does not provide a helm chart (readme redirects to prometheus-community), however under the prometheus-community's repo the k8s manifest are quite different, for example:
This repo defines a few metric relabeling for some servicemonitors: https://github.com/prometheus-operator/kube-prometheus/blob/main/jsonnet/kube-prometheus/components/k8s-control-plane.libsonnet#L236

However the helm-chart version does not include any (only the option to add): https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/exporters/kube-api-server/servicemonitor.yaml

**Comment by paulfantom** [comment, marked as answer]
Helm charts are historically based on this repository. Usually some improvements that land here are ported into helm charts, but rarely this happens the other way round.

There are additional differences between the repositories:
- helm chart repo has only basic linting tests, whereas here we use golang tests to validate the stack on last 2 kubernetes platforms. Additionally this repository is also used by OpenShift which gives us additional test coverage.
- helm chart repo does not allow easy modification of alerts and dashboards, whereas here those are "first class citizens"
- helm can be seen as more user friendly due to use of yaml, this project is using jsonnet (we are considering a switch to yaml-based config file with jsonnet 0.18.0)
